### ISSUE TITLE
Unable to publish with AssumeRolePolicy in Globals

### ISSUE THREAD

**Issue by jplock** [awslabs/aws-sam-cli#1745]
Based on https://github.com/awslabs/serverless-application-model/pull/1311, it looks like i should be able to put `AssumeRolePolicyDocument` in `Globals`, but when trying to run `sam publish`, I'm getting:

```
Publish Failed
Error: Invalid Serverless Application Specification document. Number of errors found: 1. Errors: 'Globals' section is invalid. 'AssumeRolePolicyDocument' is not a supported property of 'Function'. Must be one of the following values - ['Handler', 'Runtime', 'CodeUri', 'DeadLetterQueue', 'Description', 'MemorySize', 'Timeout', 'VpcConfig', 'Environment', 'Tags', 'Tracing', 'KmsKeyArn', 'AutoPublishAlias', 'Layers', 'DeploymentPreference', 'PermissionsBoundary', 'ReservedConcurrentExecutions', 'ProvisionedConcurrencyConfig']
Please follow the instructions in https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html
```

I'm using the aws-sam-cli 0.40.0 installed via pip

`sam validate` says the template is fine

**Comment by awood45** [comment]
Will be supported in the next SAM CLI release.

**Comment by awood45** [comment]
This was released with aws-sam-cli 0.41.0 yesterday afternoon, closing.
